src/core/utils/i18n.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import type { I18nType } from 'core/types/i18n';
+
+type GetLocalizedTextWithLinkPartsParams = {|
+  i18n: I18nType,
+  text: string,
+  linkStart?: string,
+  linkEnd?: string,
+|};
+
+export const getLocalizedTextWithLinkParts = ({
+  i18n,
+  text,
+  linkStart = 'linkStart',
+  linkEnd = 'linkEnd',
+}: GetLocalizedTextWithLinkPartsParams): Object => {
+  const linkEnds = '__LINK__';
+
+  const localizedExploreMoreLink = i18n.sprintf(text, {
+    [linkStart]: linkEnds,
```

Comment:
Hmm, for some reason I wasn't able to get this syntax to work, but it looks like it works for you, so hooray!